papaparse.js

```diff
@@ -1382,6 +1389,8 @@ License: MIT
 		var step = config.step;
 		var preview = config.preview;
 		var fastMode = config.fastMode;
+		var temp1;
+		var strictQuote = (temp1 = config.strictQuote) !== undefined ? temp1 : false;
```

Comment:
This line is a litte bit strange and hard to understand. Could you explain why it is needed?

Reply:
instead of fetching strictQuote from config once to test it for being undefined and again to use the value (which could have changed between the two points if the object installed a getter method, it saves it to a temporary local.
```
var strictQuote;

var temp1 = config.strictQuote;
if(temp1 !== undefined) {
    strictQuote = temp1;
} else {
    strictQuote = false; // default value
}
```